wsmaster/che-core-api-factory-git-ssh/src/main/java/org/eclipse/che/api/factory/server/git/ssh/GitSshUrl.java

```diff
@@ -73,7 +73,7 @@ public Optional<String> filename() {
 
       @Override
       public String location() {
-        return devfileFilename;
+        return String.format("https://%s/%s/%s", hostName, repository, devfileFilename);
```

Comment:
Sounds like significant changes.

Reply:
I do not think so. As I mentioned in the description this function is used to handle unsupported git provider and this url pattern is just potentially can match a RAW devfile url. If we handle a known provider, we have the pattern to fetch the devfile, but here we just trying to guess and with this pattern it is more likely that we will fetch the devfile.

Reply:
This change is not related to this pull request so I decided to revert it and leave a todo.